EvidenceApi/StaffSelectedDocumentTypes.json

```diff
@@ -941,13 +941,13 @@
     "documentTypes": [
       {
         "id": "vehicle-photos",
-        "title": "Photograph",
+        "title": "Photograps of vehicle",
```

Comment:
Typo in "Photograps"

Reply:
Addressed in [7167a4f](https://github.com/LBHackney-IT/evidence-api/pull/192/commits/7167a4f1f541071dc1b863c4c5d8a016c3c54033)